.github/workflows/CompatHelper.yml

```diff
@@ -3,16 +3,35 @@ on:
   schedule:
     - cron: 0 0 * * *
   workflow_dispatch:
+permissions:
+  contents: write
+  pull-requests: write
 jobs:
   CompatHelper:
     runs-on: ubuntu-latest
     steps:
+      - name: Check if Julia is already available in the PATH
```

Comment:
Why are these here?

Reply:
it is a clone of the officially recommended CH script:
https://github.com/JuliaRegistries/CompatHelper.jl/blob/master/.github/workflows/CompatHelper.yml

Reply:
Are they from a previous hot fix?

Reply:
(apparently github may provide some julia in base images)

Reply:
okay, but then this will go only later to `master` then ....